plugins/chat/app/services/chat/auto_leave_channels.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+module Chat
+  class AutoLeaveChannels
```

Comment:
We don't have test cases for this service class as well.

Reply:
Same as AutoJoin, it's being tested "implicitly" (aka. in other specs).

Reply:
Got it 👍 Thank you for clarifying .